### DISCUSSION TITLE
Black tiles after EAS deploy on iOS

### DISCUSSION THREAD

**Discussion by ludovicjamet** [rnmapbox/maps#3612] [Q&A]
Hello,

I installed the last version of the package on an Expo project. Everything is working perfectly on my local devices (iOS devices, web, simulator) but once it is deployed to the preview environment using EAS I have no tiles (but other assets are displayed. I don't get what I'm doing wrong.

I tried:
- to create a new access token
- updated the styles to suit demos
- use V11 instead of V10

Do you guys have an idea of what could be the issue?

```
     <View style={{ height: "100%", width: "100%", flex: 1 }}>
        <MapView style={{ flex: 1 }} styleURL={Mapbox.StyleURL.Street}>
          <Camera ref={cameraRef} zoomLevel={10} centerCoordinate={DefaultLocation} />
          {items?.map(
            (item, index) =>
              item.location && (
                <MarkerView
                  key={item.id}
                  coordinate={[item.location.lng, item.location.lat]}
                >
                  <View style={{ backgroundColor: "red", width: 20, height: 20, borderRadius: 10 }} />
                </MarkerView>
              )
          )}
        </MapView>
      </View>
```

**Comment by VicHofs** [comment]
I'm also having this issue!

**Comment by mfazekas** [comment]
If it's black screen it's usually access token issue. Not sure how you set your access token.
